src/lib/assets/ButtonContent.tsx

```diff
@@ -1,14 +1,14 @@
 import * as React from 'react'
 
-export default ({ children, icon }: { children: any; icon?: any }) => {
+const ButtonContent = ({ children, icon }: { children: any; icon?: any }) => {
```

Comment:
This is only used in Login.tsx; I am not happy with how it works, and I think this could be removed entirely (styling could be pushed up to the button)